src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
 
 # Output the second-to-last element: 9
-print()
+print(a[-2:-1])
```

Comment:
These are all excellent solutions 🏆 

I propose one challenge to you: that you simplify the slice notation on these two lines even further.